ALCF/helpers.sh

```diff
@@ -392,7 +405,8 @@ setParams() {
         if [[ -n "${NO_FLASH_ATTN-}" ]]; then
             echo "Not using flash-attn!!"
         else
-            LLAMA_ARGS="${LLAMA_ARGS} --use-flash-attn-builder"
+            # LLAMA_ARGS="${LLAMA_ARGS} --use-flash-attn-builder"
+            FLASH_ARG="--use-flash-attn-builder"
```

Comment:
previously, we had been including the `flash-attn` flag in with the `"${LLAMA_ARGS}"`.

since we would still like to be able to use `flash-attn` when running with `NO_LLAMA=1`, we just break it out into its own `${FLASH_ARG}` and pass this directly in our `run_cmd`